arch/x86/soc/intel_quark/quark_se/Kconfig.defconfig.series

```diff
@@ -86,6 +86,7 @@ config I2C_1
 	def_bool y
 
 if I2C_1
+
 config I2C_1_IRQ_PRI
```

Comment:
I am sure that I am missing something here. There is no interrupt property in intel,qmsi-i2c but there is IRQ_PRI here. 
Same for I2C_0.